src/app/handlers/project-web-service-handler.ts

```diff
@@ -73,5 +73,12 @@ export class ProjectWebServiceHandler {
     if (message.action === 'log.info') {
       this.infoNotificationEmitter.emit(message.event.message);
     }
+    // Controller notification
+    if (message.action === 'template.deleted') {
+      this.infoNotificationEmitter.emit(message.event.message);
+    }
+    if (message.action === 'template.updated') {
+      this.infoNotificationEmitter.emit(message.event.message);
+    }
```

Comment:
Should probably handle `template.added` as well. I will create another issue to because right now the controller notifications don't do anything excepting displayed the event in the console.